elasticsearch/connection/http_urllib3.py

```diff
@@ -116,9 +125,6 @@ def __init__(self, host='localhost', port=9200, http_auth=None,
                 'assert_hostname': ssl_assert_hostname,
                 'assert_fingerprint': ssl_assert_fingerprint,
                 'ssl_context': ssl_context,
-                'cert_file': client_cert,
-                'ca_certs': ca_certs,
-                'key_file': client_key,
```

Comment:
by leaving these here in the previous version of the SSLContext iterations, we're "half-assing" the ssl context implemetation. And as such it's causing issues. Since we are creating the SSLContext for the user if they don't pass it in, we shouldn't need to worry about passing through these variables anyway. So i'm removing them